go/libkb/passphrase_login.go

```diff
@@ -251,8 +245,31 @@ func PassphraseLoginPromptThenSecretStore(m MetaContext, usernameOrEmail string,
 	return nil
 }
 
-func StoreSecretAfterLoginWithLKS(m MetaContext, n NormalizedUsername, lks *LKSec) (err error) {
-	return StoreSecretAfterLoginWithLKSWithOptions(m, n, lks, nil)
+func LoadAdvisorySecretStoreOptionsFromRemote(mctx MetaContext) (options SecretStoreOptions) {
+	options = DefaultSecretStoreOptions()
+
+	var ret struct {
+		AppStatusEmbed
+		RandomPW bool `json:"random_pw"`
+	}
+	err := mctx.G().API.GetDecode(mctx, APIArg{
+		Endpoint:    "user/has_random_pw",
+		SessionType: APISessionTypeREQUIRED,
+	}, &ret)
+	if err != nil {
+		mctx.Warning("Failed to load advisory secret store options from remote: %s", err)
+		// If there was an API error, just return the default options.
+		return options
```

Comment:
this just indicates whether we want to store it in the file-based or try to do keyring first. I thought I made the change to default to file-based in case of errors but I didn't push it up yet, will do in next commit. Basically I didn't want the secret store to fail for some ephemeral network failure here.

Reply:
I thought the default options would trigger the file based storage